src/renderer/contexts/DependencyContext.tsx

```diff
@@ -444,25 +454,63 @@ export const DependencyProvider = memo(({ children }: React.PropsWithChildren<un
             });
     };
 
-    const installPackage = (p: Package) => {
-        setInstallingPackage(p);
+    const copyCommandToClipboard = (command: string) => {
+        clipboard.writeText(command);
+        toast({
+            title: 'Command copied to clipboard.',
+            description:
+                'Open up an external terminal, paste the command, and run it. When it is done running, manually restart chaiNNer.',
+            status: 'success',
+            duration: 9000,
+            isClosable: true,
+        });
```

Comment:
This is still displayed behind the dep manager.

Reply:
Oops yeah I'll fix that